config/jobs/kubernetes-sigs/cluster-api/cluster-api-main-periodics.yaml

```diff
@@ -275,3 +275,61 @@ periodics:
     testgrid-tab-name: capi-e2e-conformance-ci-latest-main
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+- name: periodic-cluster-api-e2e-main-latest
```

Comment:
Note: you should add the jobs to `https://github.com/SD-13/k8s-test-infra/blob/e2e_main_job/config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics.yaml.tpl` and `https://github.com/SD-13/k8s-test-infra/blob/e2e_main_job/config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-presubmits.yaml.tpl` instead and use

`TEST_INFRA_DIR=../../k8s.io/test-infra make generate-test-infra-prowjobs` to generate this file 🙂 . (note: adjust the `TEST_INFRA_DIR` according to where the repo is for you.